### ISSUE TITLE
Uniquely Identifying Windows in JS

### ISSUE THREAD

**Issue by calvinwyoung** [jigish/slate#270]
Is there any way to get a unique ID for a window in JS?  I'd like to be able to reliably distinguish between different windows of the same application.

**Comment by edkolev** [comment]
Also interested. I'd like to switch to the next window of the focused app (like cmd+`).

**Comment by DeaconDesperado** [comment]
Agreed.  It would be awesome to be able to distinguish between browser windows whose titles change very frequently.
